Core/tests/Utf8Tests.cpp

```diff
@@ -19,10 +18,8 @@ TEST_CASE("Utf8 String", "[common][utf8]") {
   SECTION("ctor & conversions") {
     gd::String str = u8"UTF8 a été testé !";
 
-    sf::String sfStr = str;
     std::u32string u32str = str.ToUTF32();
 
-    REQUIRE(str == gd::String::FromSfString(sfStr));
     REQUIRE(str == gd::String::FromUTF32(u32str));
   }
 
```

Comment:
As said in another comment, let's add some tests (check size, substr, assignement from a string to another, append a string to another) with emojis and CJK characters.